packages/squiggle-lang/__tests__/library/dist_test.ts

```diff
@@ -228,6 +228,10 @@ describe("eval on distribution functions", () => {
       "mixture(normal(5,2), normal(10,1), [0.2, 0.4])",
       "Point Set Distribution"
     );
+    testEvalToBe(
+      "mixture(normal(5,2) -> SampleSet.fromDist, normal(10,1), [0.2, 0.4])",
+      "Sample Set Distribution"
+    );
```

Comment:
It would be good if you could add another 1-3 tests here. Maybe one to show that  

"mixture(normal(5,2) -> SampleSet.fromDist, normal(10,1), [0.99, 0.1])",

Is very correlated with that first element, as we showed in the previous example.